docs/en/ingest-management/release-notes/release-notes-7.17.asciidoc

```diff
@@ -35,6 +37,23 @@ Also see:
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 7.17.8 relnotes
+
+[[release-notes-7.17.8]]
+== {fleet} and {agent} 7.17.8
+
+There are no bug fixes for {fleet} or {agent} in this release.
+
+//Review important information about the {fleet} and {agent} 7.17.8 release.
+
+//{fleet}::
+//* Add items here
```

Comment:
We need a breaking change note for fleet server, can just be a copy of the one for agent in 8.5.0: https://www.elastic.co/guide/en/fleet/current/release-notes-8.5.0.html#breaking-changes-8.5.0

This is from a security patch.

Reply:
Thanks, Craig - added that text and rebuilding the docs to check it...